tools/lint/lint.py

```diff
@@ -490,19 +520,20 @@ def check_parsed(repo_root, path, f):
         src = element.attrib["src"]
 
         def incorrect_path(script, src):
+            # type: (str, str) -> bool
             return (script == src or
                 ("/%s" % script in src and src != "/resources/%s" % script))
 
-        if incorrect_path("testharness.js", src):
+        if incorrect_path(b"testharness.js", src):
```

Comment:
Oh, it is, and everything's broken. I'll hold off on reviewing until the tests pass :)

Reply:
@Ms2ger The problem we're actually vastly more broken on Py3 than the tests suggest, and I don't know what to do about this. I don't want to block this on fixing our Py3 support (and `from __future__ import unicode_literals` makes everything worse because some things we mostly assume are `str`, like file paths, which we don't have any literal that works across Py2/3 for)… Le sigh. :/

Reply:
I'm sure that's true, but can we just revert this change to the literals for now?

Reply:
IIRC, MyPy then objects in Py2 mode because `incorrect_path`'s arguments don't allow that (and then we run into the same issue with literals within the function). I mean we can add more `type: ignore`, but I'd rather not… :/